app/js/app.js

```diff
@@ -1,49 +1,46 @@
-angular.module('cityInfo.providers', []);
-angular.module('cityInfo.factories', ['cityInfo.providers']);
-angular.module('cityInfo.controllers', ['cityInfo.factories']);
+(function() {
+  'use strict';
 
-var pw = angular.module('cityInfo', ['ngMaterial', 'cityInfo.factories', 'cityInfo.controllers']);
-pw.config(function($mdThemingProvider, LoadGoogleMapsApiProvider) {
-  $mdThemingProvider.theme('default').dark();
-  LoadGoogleMapsApiProvider.setConfig({
-    apiKey: 'AIzaSyDqNsDFc1Jz7XgdsoKWYnGyNBpZRL6PRh4',
-    libraries: ['places']
-  });
+  function configureApp($mdThemingProvider, LoadGoogleMapsApiProvider) {
+    $mdThemingProvider.theme('default').dark();
+    LoadGoogleMapsApiProvider.setConfig({
+      apiKey: 'AIzaSyDqNsDFc1Jz7XgdsoKWYnGyNBpZRL6PRh4',
+      libraries: ['places']
+    });
+  };
 
-});
+  function onAppInitialized($rootScope) {
+    $rootScope.fullscreen = false;
+    $rootScope.initialized = false;
 
-pw.run(['$rootScope', function($rootScope){
+    llb_app.addListener('window_state', function(data) {
+      $rootScope.$apply(function() {
+        $rootScope.fullscreen = data.fullscreen;
+      });
+    });
 
-  $rootScope.fullscreen = false;
-  $rootScope.initialized = false;
+    llb_app.addListener('window_dimensions', function(data) {
+      $rootScope.$apply(function() {
+        $rootScope.window_dimensions = data
+        $rootScope.fullscreen_app_dimensions = {
+          "width": data.fullscreen_width + "px",
+          //TODO Find a way around this magic number,
+          //it's not even accurate on all screen sizes
+          "height": data.fullscreen_height - 64 + "px"
+        };
 
-  llb_app.addListener('window_state', function(data){
-    if(data.fullscreen)
-    {
-      $rootScope.$apply(function(){
-        $rootScope.fullscreen = true;
-      })
-    }
-    else
-    {
-      $rootScope.$apply(function(){
-        $rootScope.fullscreen = false;
-      })
-    }
-  });
+        $rootScope.initialized = true;
+      });
+    });
+    llb_app.request('window_dimensions');
+  };
 
-  llb_app.request('window_dimensions');
-  llb_app.addListener('window_dimensions', function(data){
-    $rootScope.$apply(function(){
-      $rootScope.window_dimensions = data
-      $rootScope.fullscreen_app_dimensions = {
-        "width": data.fullscreen_width + "px",
-        //TODO Find a way around this magic number,
-        //it's not even accurate on all screen sizes
-        "height": data.fullscreen_height - 64 + "px"
-      };
+  angular.module('cityInfo.providers', []);
+  angular.module('cityInfo.services', ['cityInfo.providers'])
+  angular.module('cityInfo.factories', ['cityInfo.providers']);
+  angular.module('cityInfo.controllers', ['cityInfo.factories', 'cityInfo.services']);
```

Comment:
Uncaught Error: [$injector:modulerr] Failed to instantiate module cityInfo due to:
Error: [$injector:unpr] Unknown provider: LoadGoogleMapsApiProvider

This happens when I remove the dependencies from module declarations, and other bad stuff happens when I remove the lines completely.

So yeah, module dependencies need to be declared when creating the modules. Though I probably should change modules to match the folder-by-feature structure now...